pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -298,7 +282,7 @@ func (r *ReconcileWindowsMachineConfig) removeWorkerNodes(count int) (int, []Rec
 	// From the list of Windows VMs choose randomly count number of VMs.
 	for i := 0; i < count; i++ {
 		// Choose of the Windows worker nodes randomly
-		vm := chooseRandomNode(r.windowsVMs)
+		vm := chooseRandomNode(r.tracker.windowsVMs)
```

Comment:
Shouldn't choosing a random node also be part of the tracker?

Reply:
Good point. I made the change.